docs/reference/get-started/endpoints-keys.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[endpoints-keys]]
+== Endpoints and API keys
```

Comment:
there are going to be new better ways for accessing api-keys and cloud_id / url in the new Search Hub page, so worth investigating what the status of this is and updating this content to reflect the simpler way of getting those details.

Reply:
updated to point to search hub in https://github.com/elastic/elasticsearch/pull/103389/commits/81784a11b4f165287fdd174ede865ad175d10937